plugins/in_calyptia_fleet/in_calyptia_fleet.c

```diff
@@ -2391,15 +2388,32 @@ static int in_calyptia_fleet_init(struct flb_input_instance *in,
         create_fleet_header(ctx);
     }
 
+
+    /* If there is an uncommitted new config, delete it */
```

Comment:
It's possible there is uncommitted new config. The likely case is that the new config is not valid and can't be reloaded.

We'll keep the old config, replace the new config, and attempt to reload the new config.